src/fhir/api.test.ts

```diff
@@ -14,20 +87,12 @@ describe("FHIR resources", () => {
    * Then the fhir library should pass validation of the bundle
    */
   test('Bundle is valid', () => {
-    const patientForm = {
-      mrn: "969977",
-      firstName: "Donald",
-      lastName: "Duck",
-      dateOfBirth: "2012-03-04",
-      gender: Patient.GenderEnum.Male,
-      familyNumber: "Z409929",
-    };
 
-    const bundle = createBundle(patientForm);
+    const bundle = createBundle(testValues);
 
     const output = fhir.validate(bundle);
```

Comment:
I think I might have asked this in the last PR. But is there any reason why we are not using the `validate` method outside of the tests? Are we relying on the fact that if the test passes then we can assume the data will be valid in production?

Reply:
aha yes, we should do this for sure, make a lot of sense to ensure its valid and only post if it is

Reply:
ah and the answer here is that fhir.js only works in a node.js context which seems to be fine in the tests